pkg/apis/serving/v1/revision_defaults.go

```diff
@@ -153,7 +154,7 @@ func (rs *RevisionSpec) applyDefault(ctx context.Context, container *corev1.Cont
 	}
 }
 
-func (*RevisionSpec) applyProbes(container *corev1.Container) {
+func (*RevisionSpec) applyReadinessProbes(container *corev1.Container) {
```

Comment:
I think we can  keep the old name as the GRPC probes also touch readiness stuff.

Reply:
I renamed that one as the method just touches ReadinessProbes. It does not touch Liveness/Startup-Probes.

Reply:
I was just referring to `applyGRPCProbeDefaults` that touches readiness probes.

Reply:
Yes, `applyReadinessProbes` handles just the ReadinessProbe (but not regarding GRPC, but other stuff) and `applyGRPCProbeDefaults` handles GRPC probes, for all probe types :) But I think I know what you mean. 

Do you have a proposal for both names to make it more clear? I don't think `applyProbes` is better.

Reply:
Maybe just: applyProbesWithDefaults? If not we can merge and improve later.